deps.ts

```diff
@@ -1,26 +1,27 @@
 export {
   assert,
+  assertEquals,
   assertThrowsAsync,
 } from "https://deno.land/std@0.74.0/testing/asserts.ts";
```

Comment:
While we are at it, let's bump the dependencies as well 💪🏻 

```suggestion
} from "https://deno.land/std@0.82.0/testing/asserts.ts";
```